hivemind/p2p/p2p_daemon.py

```diff
@@ -472,9 +470,21 @@ async def add_binary_stream_handler(self, name: str, handler: p2pclient.StreamHa
         await self._client.stream_handler(name, handler)
 
     async def call_binary_stream_handler(
-        self, peer_id: PeerID, handler_name: str
+        self, peer_id: PeerID, handler_name: str, n_attempts: int = 5, delay: float = 0.5
```

Comment:
A docstring that explains that we actually retry the call multiple times would be useful IMO

Reply:
I've removed the retrying functionality here (initially, I thought it may help with passing tests, but it turned out to be not useful).